class PatientNotFoundError(Exception):
    pass


class PatientAlreadyExistsError(Exception):
    pass
